public/json/scim_v2_implementations.json

```diff
@@ -328,6 +328,14 @@ const scim_v2_implementations = {
             "developer": "Ping Identity",
             "link": "https://www.pingidentity.com/en/products/pingdatagovernance-server.html"
         },
+        {
+            "project_name": "Puzzel",
+            "client": "No",
+            "server": "Yes",
+            "open_source": "No",
+            "developer": "Puzzel",
+            "link": "https://www.puzzel.com/"
```

Comment:
Can you please update this link to your docs on SCIM support?

Reply:
I've updated with a link to Puzzel's help page now. Thanks for reviewing this.